### PR TITLE
Fix the typos and add dora code branch

### PR DESCRIPTION
**Pull request by meua**
1、change ISSUE_TEMPLATE/bug_report.md.
2、Based on the rust1.68 upgrade project dependency package version, the changes in the Cargo.lock file are all executed by the command `cargo update`
3、When the project is initially released, the version number can be 0.1 or 0.1.0
4、fix bug for issues: https://github.com/dora-rs/dora/issues/295

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -51,7 +51,7 @@ dora-coordinator = { version = "0.2.2", path = "binaries/coordinator" }
 
 [package]
 name = "dora-examples"
-version = "0.0.0"
+version = "0.1.0"
```
Comment (haixuanTao) [inline Cargo.toml:54]: We're not planning on releasing this. Please remove this change.

File: examples/python-dataflow/run.sh [hunk 2]
```diff
@@ -3,6 +3,7 @@ set -e
 python3 -m venv .env
 . $(pwd)/.env/bin/activate
 # Dev dependencies
+pip install patchelf
```
[the remark this answers is not in the document]
Comment (haixuanTao) [reply examples/python-dataflow/run.sh:6]: Actually, I saw #295 and patchelf was not installed. You can ignore this comment.

File: Cargo.toml (hunk 1 above)
Comment (meua) [reply Cargo.toml]: Rolled back to previous version number.

File: examples/python-dataflow/run.sh [hunk 3]
```diff
@@ -11,5 +12,5 @@ cd ../../../examples/python-dataflow
 # Dependencies
 pip install --upgrade pip
 pip install -r requirements.txt
-
+cargo run -p dora-cli up
```
[the remark this answers is not in the document]
Comment (meua) [reply examples/python-dataflow/run.sh]: Removed useless commands.

**Pull request merged**